bfcfg

```diff
@@ -3361,10 +3365,43 @@ print_osarg()
   fi
 }
 
+#
+# Pass some OS info to UEFI variable.
+#
+set_osinfo()
+{
+  local log_buf log_buf_len
+
+  log_buf=0x`cat /proc/kallsyms | grep -w log_buf | awk '{print $1}'`
+  [ "$log_buf" == "0x" ] && return
+  log_buf=$(printf '%016x' "${log_buf}")
+  log_buf_len=0x`cat /proc/kallsyms | grep -w log_buf_len | awk '{print $1}'`
+  [ "$log_buf_len" == "0x" ] && return
+  log_buf_len=$(printf '%016x' "${log_buf_len}")
+
+  printf "\\x06\\x00\\x00\\x00" > ${tmpdir}/os_info
```

Comment:
Q: Are we using the SetVariable() service to behave differently when writing the OsInfo variable?

Reply:
This is a dummy UEFI variable. Yes, in uefi code we'll need to handle it differently and pass the into to ATF.